lib/auth/auth_with_roles.go

```diff
@@ -983,6 +992,9 @@ func (a *ServerWithRoles) UpsertNode(ctx context.Context, s types.Server) (*type
 	if err := a.authorizeAction(types.KindNode, types.VerbCreate, types.VerbUpdate); err != nil {
 		return nil, trace.Wrap(err)
 	}
+	if s.GetScope() != "" {
+		return nil, trace.BadParameter("scoped nodes must register a control stream")
```

Comment:
nit: this will be a confusing error if a user tries to manually create a scoped node resource (which is something we will eventually need to allow once we add scoped openssh access support).  I think something a bit more generic (e.g. `UpsertNode does not yet support scoped resources`) would probably be better.